### DISCUSSION TITLE
How to check to see people started or fork?

### DISCUSSION THREAD

**Discussion by chuongmep** [community/community#23586] [New to GitHub]
<p>Hello everyone,<br>
this is a feature I’m using in previos version of github, with github current, i can’t see who started or forked, how we can enable it ? all answer are appreciated!<br>
<img src="https://user-images.githubusercontent.com/31106432/181078054-94f5dbd3-0ae2-4d60-9bc5-8bfa94d320ef.png" alt="image" data-base62-sha1="eRdO0HnHXCpu9uVGKWYXtPYDF6k" width="476" height="179"></p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>yup that’s the new UI update,</p>
<p>go to the root of the repo, at the right side, click it</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/6/06926d2cc3593a87ff1bdacf514cc2f37fdc1577.png" data-download-href="/uploads/short-url/W8A5uU9iippGrgh1b8VhMbTKt1.png?dl=1" title="Untitled"><img src="https://user-images.githubusercontent.com/47092464/181078055-aca6d4fb-4f40-4500-9f28-a9970c2ef981.png" alt="Untitled" data-base62-sha1="W8A5uU9iippGrgh1b8VhMbTKt1" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/6/06926d2cc3593a87ff1bdacf514cc2f37fdc1577_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/6/06926d2cc3593a87ff1bdacf514cc2f37fdc1577_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/6/06926d2cc3593a87ff1bdacf514cc2f37fdc1577_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/6/06926d2cc3593a87ff1bdacf514cc2f37fdc1577_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Untitled</span><span class="informations">1920×1040 90.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by chuongmep** [comment]
<p>Thanks  you , the new interface confuses me. :laughing:</p>
